packages/template/pages/[[...slug]].jsx

```diff
@@ -16,7 +17,14 @@ export default function Page({ globals, body, ...meta }) {
 
   const MDXComponents = {
     /* Head, */ // TODO why do we need this here?
-    a: CustomLink,
+    a: (props) => (
+      <CustomLink
+        data={allDocuments}
+        usehook={useMDXComponent}
+        config={siteConfig}
```

Comment:
`CustomLink` component doesn't need the whole `siteConfig` object. It only needs to know whether it should wrap a link in a `Toolip` or not. E.g.:

```suggestion
        preview={siteConfig.showLinkPreviews}
```